Games/JV.AD/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>replit</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <h1>Big Trig</h1>
+    <button onclick=startGame()>START</button>
```

Comment:
HTML attribute values should typically be put in quotes; or at least be consistent about it.  https://stackoverflow.com/questions/13056683/html-attribute-with-without-quotes
```suggestion
    <button onclick="startGame()">START</button>
```